### PR TITLE
Solve Issue 47:Get correct links into the Footer

### PR DESCRIPTION
**Pull request by yanniey**
### ISSUE/TICKET NUMBER
[ISSUE_47](Get correct links into the Footer)

### Describe the changes
Update content in the footer 

### Screenshots (if appropriate)
image

### CODE REVIEW

File: ocdaction/ocdaction/templates/layout.html [hunk 1]
```diff
@@ -47,25 +47,26 @@
...
 		        </ul>
 		    </div>
 	      <div class="col-xs-6 col-sm-4">
-	      	<h4 class="h5 global-footer__category text-uppercase">Social</h4>
+	      	<h4 class="h5 global-footer__category text-uppercase">Follow us</h4>
 		    <ul class="global-footer__list margin-top-0">
-		      <li class="global-footer__item"><a class="global-footer__link" href="#">Twitter</a></li>
-		        <li class="global-footer__item"><a class="global-footer__link" href="#">Facebook</a></li>
-		        <li class="global-footer__item"><a class="global-footer__link" href="#">Medium</a></li>
+		      <li class="global-footer__item"><a class="global-footer__link" href="http://www.twitter.com/ocdyouth">Twitter</a></li>
```
Comment (mybits) [inline ocdaction/ocdaction/templates/layout.html]: nitpicky: can we fix an indentation here?

File: ocdaction/ocdaction/templates/layout.html [hunk 2]
```diff
@@ -47,25 +47,26 @@
...
 		    </ul>
 		  </div>
 		  <div class="col-xs-12 col-sm-4">
-	      	<h4 class="h5 global-footer__category text-uppercase">Contact</h4>
+	      	<h4 class="h5 global-footer__category text-uppercase">Get in Touch</h4>
 		    <ul class="global-footer__list margin-top-0">
-		      <li class="global-footer__item"><a class="global-footer__link" href="#">Twitter</a></li>
-		        <li class="global-footer__item"><a class="global-footer__link" href="#">Facebook</a></li>
-		        <li class="global-footer__item"><a class="global-footer__link" href="#">Medium</a></li>
+		      <li class="global-footer__item"><a class="global-footer__link" href="mailto:youth@ocdaction.org.uk">Youth</a></li>
```
Comment (mybits) [inline ocdaction/ocdaction/templates/layout.html]: and here :)

File: ocdaction/ocdaction/templates/layout.html (hunk 1 above)
Comment (yanniey) [inline ocdaction/ocdaction/templates/layout.html]: Fixed in the updated PR, can you take a look again?

**Comment by yanniey** [comment]
I've updated the Youtube link and indentation issue, please check :)

**Comment by lmash** [comment]
youtube link fixed.. thanks

**Comment by mybits** [comment]
Can we please merge it, it has 2 green lights :)

**Pull request merged**